docs/whats_new/v1.0.0.md

```diff
@@ -0,0 +1,302 @@
+
+### New features
+- Support for custom forecasting metrics @shchur (#3760, #3602)
+- New forecasting metrics `WAPE`, `RMSSE`, `SQL` + improved [documentation for metrics](https://auto.gluon.ai/dev/tutorials/timeseries/forecasting-metrics.html) @melopeo @shchur (#3747, #3632, #3510, #3490)
+- Improved robustness: `TimeSeriesPredictor` can now handle data with all [pandas frequencies](https://pandas.pydata.org/docs/user_guide/timeseries.html#offset-aliases), irregular timestamps, or missing values represented by `NaN` @shchur (#3563, #3454)
+- New models: intermittent demand forecasting models based on conformal prediction (`ADIDA`, `CrostonClassic`, `CrostonOptimized`, `CrostonSBA`, `IMAPA`); `WaveNet` and `NPTS` from GluonTS; new baseline models (`Average`, `SeasonalAverage`, `Zero`)  @canerturkmen @shchur (#3706, #3742, #3606, #3459)
+- Advanced cross-validation options: avoid retraining the models for each validation window with `refit_every_n_windows` or adjust the step size between validation windows with `val_step_size` arguments to `TimeSeriesPredictor.fit` @shchur (#3704, #3537)
+
+### Enhancements
+- Enable Ray Tune for deep-learning forecasting models @canerturkmen (#3705)
+- Support passing multiple evaluation metrics to `TimeSeriesPredictor.evaluate` @shchur (#3646)
+- Static features can now be passed directly to `TimeSeriesDataFrame.from_path` and `TimeSeriesDataFrame.from_data_frame` constructors @shchur (#3635)
+
+### Performance improvements
+- Much more accurate forecasts at low time limits thanks to new presets and updated logic for splitting the training time across models  @shchur (#3749, #3657, #3741)
+- Faster training and prediction + lower memory usage for `DirectTabular` and `RecursiveTabular` models (#3740, #3620, #3559)
+- Enable early stopping and improve inference speed for GluonTS models @shchur (#3575)
+- Reduce import time for `autogluon.timeseries` by moving import statements inside model classes (#3514)
+
+### Bug Fixes / Code and Doc Improvements
+- Improve log messages @shchur (#3721)
+- Add reference to the publication on AutoGluon-TimeSeries to README @shchur (#3482)
+- Align API of `TimeSeriesPredictor` with `TabularPredictor`, remove deprecated methods @shchur (#3714, #3655, #3396)
+- General bug fixes and improvements @shchur(#3758, #3756, #3755, #3754, #3746, #3743, #3727, #3698, #3654, #3653, #3648, #3628, #3588, #3560, #3558, #3536, #3533, #3523, #3522, #3476, #3463)
+
+## EDA
+
+The EDA module will be released at a later time, as it requires additional development effort before it is ready for 1.0.
+We will make an announcement when EDA is ready for release. For now, please continue to use `"autogluon.eda==0.8.2"`.
+
+## Deprecations
```

Comment:
We use periods intermittently in this section. We should make it more consistent.

Reply:
updated